src/app/IdeasFeed.js

```diff
@@ -1,24 +1,17 @@
-import React from 'react'
-import styled from 'styled-components'
-import IdeaPost from './IdeaPost'
-
-const IdeasList = styled.div`
-  padding: 10px 10px;
-  color: white;
-  overflow: scroll;
-`
+import React from 'react';
+import IdeaPost from './IdeaPost';
 
 export default function IdeasFeed({ posts }) {
   return (
-    <IdeasList>
+    <section>
       {posts.map(post => (
         <IdeaPost
-          key={post.id}
+          key={post.title}
```

Comment:
If you write ``const {id, title, text, tags} = posts`` you destructure the props directly und don't need to write "post" every time before a prop.

Reply:
Destructuring! I like!